solidity_contracts/src/CairoPrecompiles/EthStarknetBridge.sol

```diff
@@ -0,0 +1,48 @@
+// SPDX-License-Identifier: MIT
+pragma solidity >=0.7.0 <0.9.0;
+
+import {CairoLib} from "kakarot-lib/CairoLib.sol";
+
+using CairoLib for uint256;
+
+contract EthStarknetBridge {
+    // State variable to store the owner of the contract
+    address public owner;
+
+    // Constructor sets the owner of the contract
+    constructor() {
+        owner = msg.sender;
+    }
+
+    // Modifier to restrict access to owner only
+    modifier onlyOwner() {
+        require(msg.sender == owner, "Not the contract owner");
+        _;
+    }
+
+    /// @dev The cairo contract to call
+    uint256 constant starknetEth = 0x49D36570D4E46F48E99674BD3FCC84644DDD6B96F7C741B1562B82F9E004DC7;
+    uint256 constant TRANSFER_SELECTOR = uint256(keccak256("transfer")) % 2 ** 250;
```

Comment:
Constants should be at the beginning of the contract, generally this is: 
constants
immutable var 
storage var